test/e2e/csv_e2e_test.go

```diff
@@ -33,13 +32,22 @@ import (
 	"github.com/operator-framework/operator-lifecycle-manager/test/e2e/ctx"
 )
 
-var _ = Describe("CSV", func() {
-	It("create with unmet requirements mini kube version", func() {
+var _ = Describe("CSV represents a specific version of an Operator running on a cluster", func() {
```

Comment:
Nit: I don't know that a container node description should necessarily describe the purpose of the object under test. It's sort of the subject of all its inner specs, so, IMO, it should either name a "thing" or a category of related behavior.